core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -171,7 +171,7 @@ var ExplorationEditorHistoryTab = function() {
         expect(states.length).toEqual(expectedStates.length);
         // Note: we need to compare this way because the state graph is
         // sometimes generated with states in different configurations.
-        states.forEach(function(element) {
+        states.every(function(element) {
```

Comment:
I don't think `every` is really the right choice here since `every()` is designed for evaluating a collection of booleans to see if they are all true. While this technically works, it's confusing for someone reading the code because we're not using `every()` the way it was intended to be used.

Can we use a `for` loop instead?

Reply:
Will try to implement that.